README.md

```diff
@@ -274,6 +275,29 @@ Currently we support two types of secrets, `password`s and `secureNote`s. All va
 
 For more information on 1password cli please see [here](https://support.1password.com/command-line-getting-started/)
 
+### EJSON Store
+
+[ejson](https://github.com/Shopify/ejson) is a tool for managing asymmetrically encrypted values in JSON format. This allows you to keep secrets securely in git/Github and give anyone the ability the capability to add new secrets without requiring access to the private key.
+
+First create the `production.ejson` file and store the secret value in it, then call `ejson encrypt secrets.ejson`. Then add the `ejson_file` argument to your stack in stack_master.yml:
```

Comment:
Shall we provide details about how to create this file? For example: 

```
ejson_wrapper generate --region ap-southeast-2 --kms-key-id [key_id] --file secrets/production.ejson
```

Reply:
We could add another command to stack master to create ejson files with ejson wrapper/a kms key rather than relying on the user to know about ejson wrapper and do this manually. Thoughts?

Reply:
Yeah I think that's a nice addition